### ISSUE TITLE
Implement Programs page

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#30]
Note. We will want staff to be able to edit this page content. We should probably discuss pros/cons of components before implement that instead of html content.

- [x]  finish styling
- [ ] add links

**Comment by taisdesouzalessa** [comment]
@xmatthewx here is the page. Please take a look at that and let me know your feedback in InVision.

https://invis.io/ZJ9YTAGEA#/216715364_Programs

cc: @sabrinang

**Comment by xmatthewx** [comment]
Looks good!

**Comment by gvn** [comment]
Do we have the images for this page?

**Comment by gvn** [comment]
@sabrinang Can you prep the images and put them in the Google drive?

**Comment by sabrinang** [comment]
images (.jpg 1200x900) on google drive here: https://drive.google.com/drive/folders/0B4Q8pzCpDS_eQUxzOWpWUEhTODQ?usp=sharing

**Comment by gvn** [comment]
@xmatthewx Do we have all the URLs for the links on this page?

**Comment by gvn** [comment]
@hannahkane ^

**Comment by hannahkane** [comment]
Have added links where I know them, and otherwise offered questions to consider.

 * [Mozilla Festival](https://mozillafestival.org/)
 * Global Sprints—I don't know that we'll have a destination page for the next global sprint by March 15. @stephwright to confirm?
 * Upcoming Events—not sure what was intended here, but I don't think we have a useful page for this. There are a variety of events happening (Science and Learning community calls, Hive/Gigabit events, NSF Wireless Innovation Challenge events), but we don't have a single place we can point to. 
 * Open Leadership Training—this might go to the interstitial page we're creating. See https://github.com/mozilla/network/issues/29#issuecomment-283819561
 * Work Open Workshop—the next one will have passed by the time we launch the site. @acabunoc - do we know when the first post-March 15 workshop will be?
 * [Web Literacy Activities](https://learning.mozilla.org/en-US/activities)
 * Fellowships—suggest replacing the three individual links with a single link to the Fellowships interstitial. See https://github.com/mozilla/network/issues/29#issuecomment-283819561
 * [Apply for Gigabit Grant](https://learning.mozilla.org/en-US/gigabit/apply)
 * See Grant Project Examples—I don't believe there's a single page for this (yet), but we could link to the porfolios for [Hive NYC](http://hivenyc.org/portfolio/), [Hive Chicago](http://hivechicago.org/portfolio/#/*), [Hive Toronto](http://hivetoronto.org/portfolio/), and [Gigabit cities.](https://learning.mozilla.org/en-US/gigabit/portfolio)
 * [Mozilla Gigabit Cities](https://learning.mozilla.org/gigabit)
 * [Mozilla Clubs](https://learning.mozilla.org/en-US/clubs)
 * [Hive Learning Cities](https://hivelearningnetworks.org/) (suggest copy edit "Hive Learning Networks")
 * See our [Global Research Reports](https://mozilla-foundation-research.herokuapp.com/) - @secretrobotron to verify that's the right link
 * See the [Internet Health Report](https://internethealthreport.org)—assuming we should go directly to the report, rather than the interstitial. What do you think, @xmatthewx?

Other possible things to include:
 * In Local Organizing: [Mozilla Campus Clubs](https://campus.mozilla.community/)
 * In Local Organizing: [Science Study Groups](https://science.mozilla.org/programs/studygroups)
 * In Grants: NSF Wireless Innovation Challenge (currently on hold)

**Comment by taisdesouzalessa** [comment]
> Upcoming Events—not sure what was intended here, but I don't think we have a useful page for this. There are a variety of events happening (Science and Learning community calls, Hive/Gigabit events, NSF Wireless Innovation Challenge events), but we don't have a single place we can point to.

Initially the intention was to make this link point to the Network site "Upcoming" page. However the Upcoming page evolved to not only include events but also announcements such as "Open Web Fellows Announced". 

I still think there is some value to take the users to that page because of the context - if they are interested in Events, the Upcoming page is a good place to go to see what's going on in the future. Curious to hear your thoughts on it @xmatthewx @sabrinang @hannahkane

**Comment by sabrinang** [comment]
@hannahkane What about the themes links? The only one I am unsure about is Internet of Things:

Science - http://science.mozilla.org
Women and Web Literacy - https://mozilla.github.io/womenandweb/
Internet of Things - ????
Learning - https://learning.mozilla.org/

**Comment by hannahkane** [comment]
Oh, good question, @sabrinang. We were going to fork the WWL page and create an IOT page, but we decided to do that *after* the Network site was built. I can check in with that team to see if there's something to point to in the meantime. If not, we could remove it for the time being, or perhaps have a "coming soon" indicator?

**Comment by secretrobotron** [comment]
> See our Global Research Reports - @secretrobotron to verify that's the right link

Try this link instead: mzl.la/globalweb

We'll have a DSO one soon too.

**Comment by hannahkane** [comment]
Thanks, @secretrobotron. Updated [my earlier comment](https://github.com/mozilla/network/issues/30#issuecomment-284542996) to reflect these.

**Comment by secretrobotron** [comment]
Actually, @hannahkane, mzl.la/dso already exists too.

**Comment by hannahkane** [comment]
Comment updated. Thanks, @secretrobotron.

**Comment by hannahkane** [comment]
@taisdesouzalessa - ah, right, the Upcoming page! Yeah, I think that could work, but curious what others think.

**Comment by stephwright** [comment]
Re: Global Sprint, no there will not be a destination page by March 15th.  Talking with the team at our meeting today about timeline for that.

**Comment by acabunoc** [comment]
Global Sprint Link from @auremoser https://science.mozilla.org/programs/events/global-sprint-2017

**Comment by hannahkane** [comment]
Thanks, @acabunoc. Updated the [tracking comment](https://github.com/mozilla/network/issues/30#issuecomment-284542996). We're getting closer!

**Comment by acabunoc** [comment]
For the WOW - the next one will be in June, but it's Science specific, so I don't think it should appear on this page.

We'll be adding Open Leadership 101 to Teachable, if you could like to link to that from here.

**Comment by hannahkane** [comment]
@acabunoc - that makes sense. I'll edit the comment above. Thanks!

**Comment by hannahkane** [comment]
Actually, @acabunoc, I need to clarify something. The [interstitial page copy](https://github.com/mozilla/network/issues/44#issuecomment-283819302) really only references the 101 training. So maybe on this Programs page, we'll link to that interstitial page. Is there another page we should link to that serves as a landing page for the larger Open Leadership Training?

**Comment by acabunoc** [comment]
I'm putting together a quick landing page for the larger Open Leadership Training this week! Will send you a link once it's up @hannahkane

**Comment by hannahkane** [comment]
I believe the only things we're blocked on are:
 * Open Leadership and Open Leadership 101 links (@acabunoc to provide shortly)
 * IOT link (have asked @thornet to provide)

**Comment by hannahkane** [comment]
This ticket is gnarly, and is almost 100% implemented, so I'm going to close it, and open a new ticket for the remaining tasks.
